spec/features/movies_feature_spec.rb

```diff
@@ -246,12 +257,6 @@
             create(:tagging, movie_id: counter, user_id: user.id, tag_id: tag.id)
             counter += 1
           end
-          # visit root_path
-          # visit movies_path
-          # @movie = Movie.first
-          # find("#modal_link_#{@movie.tmdb_id}")
-          # find("#modal_link_#{@movie.tmdb_id}").click
-          # click_link "hilarious"
```

Comment:
If we find we need this, we can put it back.

Reply:
yeah, let's keep it because we have an issue to fix up the failing feature specs